server/sonar-web/src/main/js/apps/quality-profiles/details/ChangeParentForm.js

```diff
@@ -55,16 +59,16 @@ export default class ChangeParentForm extends React.PureComponent {
     this.mounted = false;
   }
 
-  handleCancelClick = (event: Event) => {
+  handleCancelClick = (event /*: Event */) => {
     event.preventDefault();
     this.props.onClose();
   };
 
-  handleSelectChange = (option: { value: string }) => {
+  handleSelectChange = (option /*: { value: string } */) => {
     this.setState({ selected: option.value });
   };
 
-  handleFormSubmit = (event: Event) => {
+  handleFormSubmit = (event /*: Event */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)